pkg-manager/plugin-commands-installation/src/install.ts

```diff
@@ -296,7 +296,6 @@ export type InstallCommandOptions = Pick<Config,
   pruneDirectDependencies?: boolean
   pruneStore?: boolean
   recursive?: boolean
-  dedupe?: boolean
```

Comment:
This becomes unnecessary now that the dedupe command handler no longer calls the install command handler directly.

Removing this is technical an API break on `plugin-commands-installation`. Should I switch the changelog to be `major`?